src/main/java/org/apache/commons/collections4/bidimap/TreeBidiMap.java

```diff
@@ -782,7 +782,7 @@ private static boolean isBlack(final Node<?, ?> node, final DataElement dataElem
     }
 
     /**
-     * force a node (if it exists) red
+     * Force a node (if it exists) red.
```

Comment:
If a method starts with "make...", it usually reads better to start the Javadoc with "Makes..." IMO, It's confusing to have the Javadoc be different, otherwise you could say that all method that start with "set..." also "forces" a value.